src/reducers/requests/posts.js

```diff
@@ -46,6 +46,17 @@ function getPostThread(state = initialRequestState(), action) {
     );
 }
 
+function getPostThreadWithRetryAttempt(state = 0, action) {
+    switch (action.type) {
+    case PostTypes.GET_POST_THREAD_WITH_RETRY_ATTEMPT:
+        return state + 1;
```

Comment:
yeah good catch, probably a good idea to reset the attempt on each request since that will only be dispatched once.